### ISSUE TITLE
Description of COMPAS should be updated to sound less racist. 

### ISSUE THREAD

**Issue by amychen1776** [daviddao/awful-ai#48]
In the readme about COMPAS, it says this:

```
 The algorithm is biased against blacks (even worse than humans)
```
This implies that black people are not human.

**Comment by villasv** [comment]
It might need better wording, but the parenthesized comparison says that _the algorithm is even worse than humans_

**Comment by amychen1776** [comment]
I assume that is what it's trying to say but the order of operations reads poorly

**Comment by daviddao** [comment]
Thanks for spotting! I absolutely agree this wording can be misunderstood.

**Comment by daviddao** [comment]
Closed with 840e921
